packages/xstate-react/src/createActorContext.ts

```diff
@@ -96,7 +95,8 @@ export function createActorContext<TMachine extends AnyStateMachine>(
 
   function useActor() {
```

Comment:
Agree with removing it. Also, would this be feasible?

```ts
const value = SomeContext.useSelector(state => state.context.obj);

const { send } = SomeContext.actorRef; // no hook
```

Reply:
> Agree with removing it. Also, would this be feasible?

No, context is a singleton but we expect to get access to the instance scoped to the wrapped subtree. There is no single `SomeContext.actorRef`, that's created within the Provider.

I don't think this is that big of a problem though, it looks pretty much the same:
```diff
-const { send } = SomeContext.actorRef; // no hook
-const { send } = SomeContext.useActorRef(); // hook
```

Note that you might not want to destructure `send` like this because it might rely on `this`.